app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -18,7 +18,7 @@ class RedirectIfAuthenticated
     public function handle($request, Closure $next, $guard = null)
     {
         if (Auth::guard($guard)->check()) {
-            return redirect('/home');
+            return redirect('/board');
```

Comment:
the redirect url might be missing an 's' because the route resource is defined as '/boards'.